packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -354,11 +358,21 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
     });
   }
 
+  void scrollToHighlight() {
+    WidgetsBinding.instance.addPostFrameCallback((_) {
+      if (currentHighlight != null && buttonItemKeys[currentHighlight!].currentContext != null) {
```

Comment:
Is it possible to get here if currentHighlight is null? If so, we should test that case.

It would be nice to write this with only one lookup of the current highlight's context. Like
```dart
  void scrollToHighlight() {
    WidgetsBinding.instance.addPostFrameCallback((_) {
      if (currentHighlight == null) {
        return;
      }
      final BuildContext? highlightContext = buttonItemKeys[currentHighlight!].currentContext;
      if (highlightContext != null) {
        Scrollable.ensureVisible(highlightContext);
      );
    }
  }
```